src/routes/docs/products/functions/deploy-manually/+page.markdoc

```diff
@@ -12,32 +12,17 @@ you can also create deployments manually or through the Appwrite CLI.
 
 # CLI  {% #cli %} 
 
-{% info title="CLI setup" %}
-Before you can deploy with the Appwrite CLI, make sure you've [installed and initialized](/docs/tooling/command-line/installation) the CLI.
-{% /info %}
+Before deploying your function and updating, deleting, or downloading its deployment using the CLI, [**install**](/docs/tooling/command-line/installation#Installation) the CLI, [**log in**](/docs/tooling/command-line/installation#get-started) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#Initialization) your Appwrite project. Use the CLI commands or edit the `appwrite.json` file to create, update, list, and delete variables.
 
-To deploy with the Appwrite CLI, your function must be added to `appwrite.json`. 
-Use the `appwrite init function` method to create a starter function, then paste in your function code.
-
-```bash
-appwrite init function
-```
-
-Edit the generated code, add dependencies, and deploy the function using the following command:
-
-```bash
-appwrite deploy function
-```
-
-## Overwrite warning {% #overwrite-warning %} 
-
-If you made changes in the Appwrite Console that are different from your `appwrite.json`, using the CLI deploy command will overwrite your console changes. Update your `appwrite.json` manually before deploying.
+{% arrow_link href="/docs/tooling/command-line/functions#commands" %}
+Learn more about the CLI functions commands
+{% /arrow_link %}
```

Comment:
This new bit is unclear and doesn't really explain deploying functions using the CLI